### PR TITLE
Adds Catnip plants and tea

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
Adds a chem that makes you more feline!
Adds catnip to get this chem
Adds catnip sleepy time tea - The best tea

## Why It's Good For The Game

Nya~ Purrfect chem for kittys and doggos alike

## Changelog
:cl:
add: amazing things like tea of catnip, catnip and plant
/:cl:

### CODE REVIEW

**Comment by cebutris** [comment]
The catnip chem could just be a subtype of moonsuagar with a different taste

EDIT: Nevermind I'm a dumbass it's already a subtype of pax

**Comment by davidchan13** [comment]
Some do some follow up code to make runtime go apeshit around a harvestable plant or leaves of catnip.

**Comment by Trilbyspaceclone** [comment]
@davidchan13 Wish I was smarts and able to do that but atm cant really

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1962,3 +1962,15 @@
...
+	name = "catnip"
+	id = "catnip"
+	taste_description = "grass"
+	description = "A colorless liquid that makes people more peaceful and felines more happy."
+	metabolization_rate = 1.75 * REAGENTS_METABOLISM
+
+/datum/reagent/pax/catnip/on_mob_life(mob/living/carbon/M)
+	M.emote("nya")
+	to_chat(M, "<span class = 'notice'>[pick("Headpats feel nice.", "The feeling of a hairball...", "Backrubs would be nice.", "Whats behind those doors?")]</span>")
+	M.adjustArousalLoss(2)
```
Comment (Thalpy) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Does this not need ..() to call the proc of pax?

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 2]
```diff
@@ -881,3 +881,20 @@
...
+	taste_description = "sugar and catnip"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of catnip tea"
+	glass_desc = "A purfect drink for a cat."
+
+/datum/reagent/consumable/catnip_tea/on_mob_life(mob/living/carbon/M)
+	M.emote("nya")
+	to_chat(M, "<span class = 'notice'>[pick("Headpats feel nice.", "Backrubs would be nice.", "Mew")]</span>")
+	M.adjustArousalLoss(5)
+	M.adjustStaminaLoss(3)
```
Comment (Thalpy) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Kev please comment on this.

**Review by Thalpy — COMMENTED** [review]
Overall I like it! When it's merged I'll add catnip as an alternative reagent for furranium as a PR.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 2 above)
Comment (Thalpy) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: @kevinz000 I forgot I can do this. Specifically stamina loss.

Comment (Trilbyspaceclone) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: sleepy tea!

**Review by silicons — CHANGES_REQUESTED** [review]
please cap the catnip tea staminaloss so it doesn't stamcrit people.
i'm also semi-negative on the idea of botany being able to grow literal pax, even if short-lasting, because of how op botany is these days @BlackMajor @deathride58

**Comment by Trilbyspaceclone** [comment]
how do I cap that?

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 3]
```diff
@@ -881,3 +881,23 @@
...
+/datum/reagent/consumable/catnip_tea
+	name = "Catnip Tea"
+	id = "catnip_tea"
+	description = "A sleepy and tasty catnip tea!"
+	color = "#101000" // rgb: 16, 16, 0
+	nutriment_factor = 0
+	taste_description = "sugar and catnip"
+	glass_icon_state = "teaglass"
+	glass_name = "glass of catnip tea"
+	glass_desc = "A purfect drink for a cat."
```
Comment (Thalpy) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Should be "A purrfect drink for a cat."

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm [hunk 4]
```diff
@@ -881,3 +881,23 @@
...
+
+/datum/reagent/consumable/catnip_tea/on_mob_life(mob/living/carbon/M)
+	M.getStaminaLoss(min(50 - M.getStaminaLoss(), 3))
```
Comment (Thalpy) [inline code/modules/reagents/chemistry/reagents/drink_reagents.dm]: Should this be
M.adjustStaminaLoss(min(50 - M.getStaminaLoss(), 3))?

**Review by Thalpy — COMMENTED** [review]
Cutes.

File: code/modules/reagents/chemistry/reagents/drink_reagents.dm (hunk 3 above)
Comment (Thalpy) [reply code/modules/reagents/chemistry/reagents/drink_reagents.dm]: I mean, if you like cat puns too.

**Comment by lolman360** [comment]
@kevinz000 botany can already make all the precursors to pax though

**Comment by nik707** [comment]
@kevinz000

**Review by silicons — APPROVED** [review]

**Pull request merged**